binr/preload/libr2.c

```diff
@@ -37,8 +37,10 @@ static void sigusr2(int s) {
 static void _libwrap_init() __attribute__ ((constructor));
 static void _libwrap_init() {
 	char *web;
-	signal (SIGUSR1, sigusr1);
-	signal (SIGUSR2, sigusr2);
+	sigset_t mask;
+	sigemptyset (&mask);
+	r_sys_sigaction (SIGUSR1, sigusr1, &mask, 0);
+	r_sys_sigaction (SIGUSR2, sigusr2, &mask, 0);
```

Comment:
just use vararg and make this RsysSigAction mask = r_sys_sigaction (SIGUSR2, sigusr2, 0, SIGUSR1, sigusr1, 0, -1);`

also, is the `0` really necessary? can we just skip that argument from the sigaction function definition?